cassandra-oss/Dockerfile.ubi7

```diff
@@ -0,0 +1,183 @@
+# Copyright DataStax, Inc, 2020
+#   Please review the included LICENSE file for more information.
+#
+ARG BASETAG=7.8
+FROM registry.access.redhat.com/ubi7/ubi-minimal:${BASETAG} AS builder
+
+ARG URL_PREFIX=https://dlcdn.apache.org/cassandra
+ARG CASSANDRA_VERSION=trunk
```

Comment:
This isn't used, but probably should be on line 115 below. It's set to `trunk` here because that's what it's going to build. I probably should adjust the git clone steps to checkout the branch that matches this version, that way it could be used to build any branch from the repo